packages/contentful/migrations/0083.1-delete-heroHeader-content-type-entries.cjs

```diff
@@ -0,0 +1,5 @@
+const deleteContentEntries = require('../src/utils/deleteContentEntries.cjs');
+
+module.exports = async function (migration, context) {
+  await deleteContentEntries('heroHeader', migration, context);
+};
```

Comment:
Could these migrations contain both the entries and content type deletion? Like so:
```
const deleteContentEntries = require('../src/utils/deleteContentEntries.cjs');

module.exports = async function (migration, context) {
  await deleteContentEntries('heroHeader', migration, context);

  migration.deleteContentType('heroHeader');
};
```

Reply:
Yes, thanks, that makes more sense, and has been actioned.